src/esp/sensor/CameraSensor.cpp

```diff
@@ -19,6 +19,17 @@ CameraSensor::CameraSensor(scene::SceneNode& cameraNode,
     : VisualSensor(cameraNode, spec),
       baseProjMatrix_(Magnum::Math::IdentityInit),
       zoomMatrix_(Magnum::Math::IdentityInit) {
+  // Sanity check
```

Comment:
Sanity check should be done on each level, not just the CameraSensor. For example, it should be done in the VisualSensor as well.